README.md

```diff
@@ -1,222 +1,166 @@
 
-## Linting Assets (JS,SASS)
 
-Lints JS and SASS using Webpack
+3. You have BrowserSync to sync assets and enable easy cross-device testing. Once BrowserSync is active you will get url in the terminal. Usualy it is `localhost:3000`.
 
-```bash
-npm run precommit
-```
+## Liniting Files
 
-## Linting PHP ##
+1. **Linting assets** - JS and SASS using Webpack:
 
-We are using [Infinum coding standards for WordPress](https://github.com/infinum/coding-standards-wp) to check php files.
+    ```bash
+    npm run precommit
+    ```
 
-To install it, you need to install [Composer](https://getcomposer.org/) first.
+2. **Linting PHP** - We are using [Infinum coding standards for WordPress](https://github.com/infinum/coding-standards-wp) to check php files. To install it, you need to install [Composer](https://getcomposer.org/) from initial setup step. 
 
-* Add this aliases to you bash config:
+3. **Linting PHP** - If you have composer installed add this aliases to you bash config and reload terminal:
 
-```bash
-alias phpcs='vendor/bin/phpcs';
-alias phpcbf='vendor/bin/phpcbf';
-alias wpcs='phpcs --standard=vendor/infinum/coding-standards-wp/Infinum';
-alias wpcbf='phpcbf --standard=vendor/infinum/coding-standards-wp/Infinum';
-```
-* Reload terminal
+    ```bash
+    alias phpcs='vendor/bin/phpcs';
+    alias phpcbf='vendor/bin/phpcbf';
+    alias wpcs='phpcs --standard=vendor/infinum/coding-standards-wp/Infinum';
+    alias wpcbf='phpcbf --standard=vendor/infinum/coding-standards-wp/Infinum';
+    ```
 
-Checking theme for possible violations:
+4. **Linting PHP** - Here are available scripts to run in your terminal to lint your PHP files:
+    
+    Checking theme for possible violations:
 
-```bash
-wpcs wp-content/themes/inf_theme
-```
+    ```bash
+    wpcs wp-content/themes/inf_theme
+    ```
 
-Autofix theme for minor violations:
+    Autofix theme for minor violations:
 
-```bash
-wpcbf wp-content/themes/inf_theme
-```
+    ```bash
+    wpcbf wp-content/themes/inf_theme
+    ```
 
-## Build
+## Goint to Production
 
-Build creates public folder in theme all the assets.
+You can do it manualy using the command from package.json but we prefere using continious integration and we have 2 scripts to help you.
 
-Check for errors js, css, php but not WP standards
+1. Run build script to build production version of assets and composer, run linting and tests:
 
-```bash
-bash bin/prebuild.sh
-```
+    ```bash
+    bash bin/build.sh
+    ```
 
+2. Run build script to build production version of assets and composer.
 Builds production ready assets
 
-```bash
-bash bin/build.sh
-```
+    ```bash
+    bash bin/build.sh
+    ```
+
+3. When rsyncing to server use `ci-exclude.txt` to exclude unnecesery folders and files on server.
```

Comment:
Add a phrase: 'For deployment, we are using [rsync](https://rsync.samba.org/) in ore continuous deployment setup', and maybe add a sample example.